Header.h

```diff
@@ -5,5 +5,23 @@
 // Ayisha Sameera,GET.
 // ------------------------------------------------------------------
 // Header.h
-// Program on branch main.
+// Program on branch A5.
+// Assignment 5 : Implementation of heap sort and binary search
 // ------------------------------------------------------------------------------------------------
+#define YELLOW "\033[1;33m"        // Color codes
+#define MAGENTA "\033[1;35m"
+#define CYAN "\033[1;36m"
+#define BLUE "\033[1;34m"
+#define RESET "\033[0m"
+
+/// <summary>Changes the array to max heap format.</summary>
+void Heapify (int arr[], int length, int head);
+
+/// <summary>Recursively deletes the max element from max heap.</summary>
+void HeapSort (int arr[], int length);
+
+/// <summary>Finds the index position of an element in the sorted array.</summary>
+int BinarySearch (int arr[], int high, int num);

```

Comment:
Since Binary Search expects sorted input, you should name your param to sortedArr to reflect that. Also, length is a more appropriate param name for this. You may be using it as the initial high but that is something for you to know, not the user.